Pull Requests/README.md

```diff
@@ -0,0 +1,64 @@
+
+Reviewing PRs is a routine task for every member of the team each work day. Because of this, we should always aim to keep our PRs as concise and descriptive as possible. Doing so allows our teammates to more quickly get up to speed with the work we are doing.
+
+## Guidelines
+
+At a minimum, a PR should have a description, a list of changes made, and the link(s) to the connected Clubhouse stories.
+
+### Description
+
+The description section should briefly describe the PR and what it aims to accomplish.  For instance, say you are working on a story that adds a section to a registration form that requests the user's age.  A description would include the goal of the PR: to add a section in the registration form for age verification.
+
+**Note:** In most cases, this will most likely be similar if not the same as the clubhouse story description.
+
+### Changes Made
+
+The changes made section should include any information about what was changed in the PR, including code refactors, schemas, or requirement changes.  For instance, say you are working on a backend route that uses a singleton pattern for a class you need to complete your story.  While you are working on the story, you end up having to change the pattern for that class to a factory pattern and the database schema needs to change.  In your PR, you should include these changes so reviewers can review if they were necessary.
+
+### Clubhouse Stories
+
+The Clubhouse stories section should include any stories that are associated with the PR.  This should include at least the story number and the associated link.  The goal of this section is to allow reviewers to see the stories it's associated with for more information that is not present in the PR description and changes made sections (as some PRs can become quite large and involve multiple stories).
+
+While not required, it is helpful to include a link to the PR created on the clubhouse story as well.
+
+### Re-Requesting Reviews
+
+Most PRs you author will most likely have a 'changes requested' review at some point, especially if the story you are working on is a larger, complex story.  In these cases, you should always re-request the person who asked for changes and also re-request anyone who has commented and already approved the PR.  This is necessary because, for some changes, the changes requested could involve a large refactor of what you have already written.  In this case, the code that was already approved and reviewed has changed significantly and should be re-reviewed because of those changes.
```

Comment:
"Most PRs you create will have a 'changes requested' review, especially if the requirements of the associated story are more complex." " and also re-request anyone who has reviewed, commented or approved your PR".